examples/docker/docker-compose-full.yml

```diff
@@ -29,21 +29,47 @@ services:
       - GF_AUTH_ANONYMOUS_ORG_ROLE=Admin
       - GF_AUTH_DISABLE_LOGIN_FORM=true
     depends_on:
+      - prometheus
       - tempo
+
+  otelcol:
```

Comment:
Should we add these to the auxiliary file too?

Reply:
Probably yes, I'll add it there as well 👍